packages/rich-text-editor/src/vaadin-rich-text-editor-mixin.js

```diff
@@ -424,6 +424,37 @@ export const RichTextEditorMixin = (superClass) =>
           this.$.linkUrl.focus();
         });
       });
+
+      // Set up tooltip to show when hovering or focusing toolbar buttons
+      this._tooltip = document.createElement('vaadin-tooltip');
+      // Create dummy aria target, as toolbar buttons already have aria-label, and also cannot be linked with the
+      // tooltip being in the light DOM
+      this._tooltip.ariaTarget = document.createElement('div');
+      this.append(this._tooltip);
```

Comment:
It probably makes sense to also add a slot for the tooltip e.g. `<slot name="tooltip">` to the template, like we do in other components. It works without, apparently because there are no slots in RTE. But we'll also need to move confirm-dialog to the light DOM, so having a dedicated slot makes sense.

Reply:
Added the slot.